docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,203 @@
+# Proposal A - FreeBSD Jails
+
+This proposal recommends changes to describe the requirements for mapping an OCI
+runtime container to a corresponding FreeBSD jail.
+
+## Modifications
+
+This suggests adding an object to the [FreeBSD-specific section](https://github.com/opencontainers/runtime-spec/blob/main/config.md#platform-specific-configuration) of the [container configuration](https://github.com/opencontainers/runtime-spec/blob/main/config.md) to describe the required parameters for the jail.
+
+## Jail Configuration
+
+Jail parameters and devfs rules for the container's jail
+
+**`devices`** _(array of object, OPTIONAL)_ - devfs rules for this container.
+
+Each element is an object with the following fields:
+
+- **`path`** _(string, REQUIRED)_ - the device path relative to "/dev"
+- **`mode`** _(string, OPTIONAL)_ - the device permissions
+- **`unhide`** _(bool, OPTIONAL)_ - if set, expose the path in the container
```

Comment:
If you add an entry for some device without setting unhide, it would hide the device. I can't think of why this would be used in this context - the default contents of /dev are close to the minimum set needed for normal operation.

This part of the proposal came from the devfs(8) manpage but it might be better to remove the 'unhide' field here and make it clear that listing a device in this section will add the device, i.e. make unhide implicitly true.

Reply:
I changed this section to remove the confusing unhide parameter and to change the mode to an integer